scripts/02-create-compose-env.sh

```diff
@@ -77,6 +77,12 @@ NVIDIA_VISIBLE_DEVICES=${NVIDIA_VISIBLE_DEVICES:-$(select_version "Select which
 BUILD_TESTS=${BUILD_TESTS:-$(select_version "Select whether to configure to build RAPIDS tests (ON/OFF)" "ON")}
 BUILD_BENCHMARKS=${BUILD_BENCHMARKS:-$(select_version "Select whether to configure to build RAPIDS benchmarks (ON/OFF)" "ON")}
 
+if [[ "$CUDA_VERSION" != "11.2.0" ]]; then
+  CONDA_CUDA_TOOLKIT_VERSION="$CUDA_VERSION"
+else
+  CONDA_CUDA_TOOLKIT_VERSION="11.2"
+fi
+
```

Comment:
seems like we'd want this to be the `major.minor` of whatever version we're going to use in the container, right?
```suggestion
CONDA_CUDA_TOOLKIT_VERSION=$(echo $CUDA_VERSION | cut -d'.' -f1,2)

```